content/kubermatic/master/installation/install_kubermatic_ee/_index.en.md

```diff
@@ -1,14 +1,16 @@
 +++
 title = "Install Kubermatic Kubernetes Platform (KKP) EE"
 date = 2018-04-28T12:07:15+02:00
-weight = 20
+weight = 30
 enterprise = true
 +++
 
-This chapter explains the installation procedure of KKP into a pre-existing Kubernetes cluster.
+This chapter explains the installation procedure of KKP Enterprise Edition (EE) into a pre-existing
```

Comment:
Are we going with "chapter" or "document"?

Reply:
It's inconsistent throughout our entire docs, unfortunately. I think we also start documents randomly with a h1, h2 or h3, whatever "looked better".

I would welcome consistency, but it's more work than this PR should do.